src/jdk.incubator.foreign/share/classes/jdk/incubator/foreign/CLinker.java

```diff
@@ -126,6 +129,20 @@ static CLinker getInstance() {
         return SharedUtils.getSystemLinker();
     }
 
+    @CallerSensitive
+    public static Optional<MemoryAddress> findNative(String name) {
```

Comment:
Security manager checks should be removed here. Also, the method is missing javadoc.

Reply:
Will remove security manager checks and add javadoc comment